### DISCUSSION TITLE
Add support for find and replace

### DISCUSSION THREAD

**Discussion by NekrosBurek** [pelican-dev/panel#441] [Ideas]
I don't know if this is changeable inside the File Editor, but if possible this could be the one of the most useful features.

At the moment while searching inside the file editor we can find multiple files and mark them it would be awesome allowing us to have the option to find and replace one or all values inside the file manager.

Haven't checked but if it does not support regex it would be awesome as well to support that inside search option

**Comment by Boy132** [comment]
You can already do find & replace by pressing `Ctrl` + `Shift` + `F` (or  `Ctrl` + `Shift` + `R` for "replace all"). And it also supports regex just like the normal search.

For all keybinds see: https://codemirror.net/5/demo/search.html

**Reply by rmartinoscar** [reply]
Maybe add a way to change those bindings or prevent default behaviour cause Ctrl + (Shift) + F works but Ctrl + (Shift) + R just refreshs the page (had same issue with Pterodactyl)
Tried it on Mozilla Firefox, Chrome, Safari (Only one working for me), Opera, Brave.
